src/model/class.ts

```diff
@@ -122,8 +121,8 @@ export class Class implements Feature {
   }
   readonly jsdoc: jsdocLib.Annotation|undefined;
   description: string;
-  readonly properties: Property[] = [];
-  readonly methods: Method[] = [];
+  readonly properties = new Map<string, Property>();
+  readonly methods: Map<string, Method> = new Map();
```

Comment:
these two are defined differently? (methods with typedef on left and properties typedef is on the value) any particular reason?